documentation/context.md

```diff
@@ -10,10 +10,12 @@ for multiple operations as much as possible.
 Initializers
 ------------
 
-#### new
+#### new(random_bytes: nil)
 
 Returns a newly initialized libsecp256k1 context. The context is randomized at
-initialization.
+initialization. The `random_bytes` argument can optionally take a string
```

Comment:
A better name for this would probably be something like `context_randomization_bytes`. A class method should also be added context, perhaps something like `Context.create_randomized` that handles passing in the 32-bytes of data from `SecureRandom` for users.